packages/terra-clinical-onset-picker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Clear existing data when granularity is changed.
```

Comment:
It'd be great if you could briefly summarize all the changes that have been made as part of this PR in the PR description.

Reply:
Added in 1417f94